storage/store/sql/sql.go

```diff
@@ -137,6 +137,18 @@ func (s *Store) GetEndpointStatusByKey(key string, params *paging.EndpointStatus
 	return endpointStatus, err
 }
 
+func (s *Store) GetGroupAndNameByKey(key string) (string, string, error) {
+	tx, err := s.db.Begin()
+	if err != nil {
+		return "", "", err
+	}
+	_, group, name, err := s.getEndpointIDGroupAndNameByKey(tx, key)
+	if err == nil {
+		return group, name, nil
+	}
+	return "", "", err
```

Comment:
```suggestion
	_, group, name, err := s.getEndpointIDGroupAndNameByKey(tx, key)
	if err != nil {
		return "", "", err
	}
	return group, name, nil
```

Reply:
Actually, this may not be needed after all. See https://github.com/TwiN/gatus/pull/267#discussion_r847819551.